robotbase/src/main/java/com/team2357/frc2022/subsystems/TurretSubsystem.java

```diff
@@ -133,6 +133,19 @@ public void periodic() {
         SmartDashboard.putNumber("Turret Motor Output", m_turretMotor.getAppliedOutput());
     }
 
+    /**
+     * Determines what direction the turret needs flipped
+     * @return Clockwise or CounterClokwise soft limit
+     */
+    public double getTurretFlipRotations() {
+        if (m_targetMotorRotations - m_config.m_turretRotationsClockwiseSoftLimit > 
+            m_targetMotorRotations - m_config.m_turretRotationsCounterClockwiseSoftLimit) {
+           return m_config.m_turretRotationsClockwiseSoftLimit;
```

Comment:
Why is this returning the soft limit? Shouldn't it be returning a value plus or minus the number of motor rotations that equals 360 degrees

Reply:
I fixed this.